examples/withSizeClass.js

```diff
@@ -54,6 +54,22 @@ export default () => (
           <ResponsiveButton title="First button" />
           <ResponsiveButton title="Second button" />
           <ResponsiveButton title="Third button" />
+          <WithSizeClass>{
```

Comment:
It might make more sense to extract this into component `ResponsiveButtonFaCC` and change the other one to the `ResponsiveButtonHOC`, that way it would be clear those components are the same just different syntax. I would prefer to avoid inlining component directly like this, makes example less readable.